azerothcore/azerothcore-wotlk
# fix error when execute vmap4assembler.exe in extractor.bat

## Changes Proposed:
When using apps\extractor\extractor.bat batch file to generate all data, there is an error when vmap4assembler.exe is executed because there are no additional parameters

## Issues Addressed:
Add parameters to the vmap4assembler.exe program in the batch file(there is another way to imitate TrinityCore's practice, pass in the default parameters in the source code of vmap4assembler.exe), here I simply add the missing parameters to the batch file

## SOURCE:


## Tests Performed:
Win10 x64
1. Copy the mapextractor.exe, mmaps_generator.exe, vmap4assembler.exe, vmap4extractor.exe generated by the program to 3.3.5 the directory where the client is located (you need to check the TOOLS option in CMAKE)
2. Copy apps\extractor\extractor.bat of the azerocore code library to 3.3.5 the directory where the client is located
3. Execute extractor.bat, enter option 4, Extract all (may take hours)
![1627879632(1)](https://user-images.githubusercontent.com/2327445/127805604-ccfa85d1-9e85-463d-9660-f16584da586a.png)

## How to Test the Changes:
Refer to Tests Performed

## Known Issues and TODO List:
None


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
